examples/activities/resample.js

```diff
@@ -23,21 +23,29 @@ module.exports = function resample(localInputPath, outputUri, options, callback)
       return callback(err);
     }
 
+    // TODO: Shouldn't assume target SRS.
     var args = [
       "-q",
-      "-t_srs", "EPSG:3857",
+//      "-t_srs", "EPSG:3857",
       "-te", options.targetExtent[0], options.targetExtent[1], options.targetExtent[2], options.targetExtent[3],
-      "-tr", options.targetResolution[0], options.targetResolution[1],
-      "-tap",
+//      "-tr", options.targetResolution[0], options.targetResolution[1],
+//      "-tap",
+      "-ts", "256", "256",
```

Comment:
@mojodna  There's changes to this file that I will revert since they are sort of specific to the OAM use case (and now exist as an activity in the `oam-server` repo), but I wanted you to see them first.

I found when using `tap` that I would get that weird shrinking tile effect we talked about. Instead, setting the `-ts` was giving me good results.